content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst

```diff
@@ -11,8 +11,16 @@ Users can link third-party shipping carriers to Odoo databases, in order to veri
 delivery to specific addresses, :doc:`automatically calculate shipping costs <delivery_method>`, and
 :doc:`generate shipping labels <../advanced_operations_shipping/labels>`.
 
-In Odoo, shipping carriers can be applied to a sales order (SO), invoice, or delivery order (DO).
-The following is a list of available shipping connectors:
+In Odoo, shipping carriers can be applied to a sales order (SO), invoice, or delivery order. For
+tips on resolving common issues when configuring shipping connectors, :ref:`skip to this section
+<inventory/shipping_receiving/third-party-troubles>`.
```

Comment:
Adding a little more meaning into the link label, so it is clear to the user where they will end up after clicking the link :)
```suggestion
tips on resolving common issues when configuring shipping connectors, skip to the :ref:`Troubleshooting
<inventory/shipping_receiving/third-party-troubles>`  section.
```

Reply:
This is really good advice! I'll make sure all the link labels are named, from now on!